src/media/local-stream.ts

```diff
@@ -130,7 +130,7 @@ abstract class _LocalStream extends Stream {
    */
   async disposeEffects(): Promise<void> {
     // Dispose of any effects currently in use
-    if (this.effects.length > 0) {
+    if (this.loadingEffects.size > 0) {
```

Comment:
I think we do want this to be `this.effects`?

Reply:
Updated.